test/spec.js

```diff
@@ -499,19 +499,6 @@ export default [
       },
     },
   },
-  {
```

Comment:
Why did you remove this test?

Reply:
If I understood well, it's doing exactly the same thing as 2 tests above (report unused bin dependencies if ignoreBinPackage equal false).

Reply:
ooooh good catch!